tests/source/hex_literal_upper.rs

```diff
@@ -0,0 +1,5 @@
+// rustfmt-hex_literal_case: ToUpper
+fn main() {
+    let h1 = 0xCAFE_5EA7;
+    let h2 = 0xCAFE_F00Du32;
```

Comment:
Since this is a case to highlight the letter casing being changed to upper I'd like a more lower-case centric input to make it easier on the eyes

e.g. 
```suggestion
    let h1 = 0xCaFE_5ea7;
    let h2 = 0xCAFE_F00Du32;
```